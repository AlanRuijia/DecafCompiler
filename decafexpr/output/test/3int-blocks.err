output prefix: /Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks
llvmas: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as
llc: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc
cc: gcc
codegen: /Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/answer/decafexpr
stdlib: /Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/answer/decaf-stdlib.c
generating llvm code:/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/answer/decafexpr ... ok
; ModuleID = 'C'

define i32 @main() {
entry:
  %main = alloca i32
  %y = alloca i32
  %x = alloca i32
  %q = alloca i32
  %p = alloca i32
  %y1 = alloca i32
  %x2 = alloca i32
}

define i32 @main.1() {
entry:
  ret i32 0
}
assembling to bitcode:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm" -o "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.bc" ... failed (1)
/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as: /Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm:12:1: error: expected instruction opcode
}
^
converting to native code:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.bc" -o "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.s" ... ok
linking:gcc -o "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.exec" "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.s" "/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/answer/decaf-stdlib.c" ... ok
running:/Users/Jerry/Develop/SFU/compiler/DecafCompiler/decafexpr/output/test/llvm/3int-blocks.llvm.exec ... ok
