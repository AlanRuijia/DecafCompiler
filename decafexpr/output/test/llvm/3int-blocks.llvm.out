Debug message: Generating extern function...
Arg Type: 17 
Inserting method decl into symbol table...
Inserting block into symbol table...
Inserting block into symbol table...
Inserting block into symbol table...
Inserting block into symbol table...
Debug message: Generating program...
Debug message: Generating package...
Debug message: Generating FieldDecls...
Debug message: Generating MethodDecl...
Codegen for Var definition...
Codegen for Var definition...
Codegen for Var definition...
Codegen for Var definition...
Codegen for Var definition...
Defind variable in line 4 : x
Defind variable in line 8 : y
Debug message: Generating method call... Calling print_int
here1
Codegen for RValue x
Success in RValue
